src/target/riscv/riscv-013.c

```diff
@@ -686,6 +715,25 @@ static int dmi_write_exec(struct target *target, uint32_t address,
 	return dmi_op(target, NULL, NULL, DMI_OP_WRITE, address, value, true, ensure_success);
 }
 
+static bool dbgbase_verify(struct target *target)
+{
+	uint32_t dbgbase_set = target->dbgbase;
+	uint32_t nextdm = 0;
+
+	while (1) {
```

Comment:
If you like, you can add more prints to make the log analysis easier.

```suggestion
	LOG_TARGET_DEBUG(target, "Searching for DM with DMI base address (dbgbase) = 0x%x", target->dbgbase);
	while (1) {
```